src/modules/specializations/specializations.resolver.ts

```diff
@@ -42,4 +57,13 @@ export class SpecializationsResolver {
   ): Promise<boolean> {
     return this.specializationsService.delete(specializationId);
   }
+
+  @ResolveField('doctors')
+  async getDoctor(@Parent() specialization: Specialization): Promise<Doctor[]> {
+    return (
+      await this.specializationsService.findOneById(specialization.id, {
+        relations: ['doctors'],
+      })
+    ).doctors;
```

Comment:
Why don't you just query doctors by specialization id?

Reply:
There is an extra task, to create FieldResolver for specialization.
Or what did you mean?

Reply:
I meant that you can inject the doctor service and query doctors with specializationId filtration. No need to query specialization as you don't use it at all. 
(Not critical)